packages/widget/src/components/network-selector/styles.ts

```diff
@@ -6,6 +6,7 @@ export const styles = css`
     border: 1px solid #e4e4e7;
     display: flex;
     width: 314px;
```

Comment:
should we change all px to rem?

Reply:
Yes, we should stick to rem as they are widely used in Figma. Also, we should avoid the exact values for the width and height and use max-width and max-height instead.

Reply:
I guess in the scope of the PR and the component, all units that are in px should be change them then